spec/controllers/announcements_controller_spec.rb

```diff
@@ -55,6 +56,59 @@
     end
   end
 
+  describe "GET /coronavirus/:coronavirus_page_slug/announcements/:id/edit" do
+    before do
+      stub_user.permissions << "Unreleased feature"
```

Comment:
I think we should a test or two to make sure user can't access the page if they don't have the right permissions.

Reply:
Done :)